### ISSUE TITLE
Scrollbar Appears and Buttons Don't Reposition Immediately on Window Resize

### ISSUE THREAD

**Issue by SPluzh** [Vaei/FloatingShelf#2]
When resizing the window to a smaller size, a scrollbar appears, but the buttons on the floating shelf do not immediately reposition themselves to fit within the new window dimensions. This causes a temporary layout issue where buttons overlap or extend beyond the visible area until further adjustments are made.
![Maya_jewXOCQ0sC](https://github.com/user-attachments/assets/5679f266-34e9-40fa-9a3d-ba7436c883c7)

**Comment by SPluzh** [comment]
This issue might be related to the display scale settings in Windows. For example, I am using a 150% scale, which could be affecting the layout behavior.

**Comment by Vaei** [comment]
I can't repro this. Did you test without the 150% scale?

**Comment by SPluzh** [comment]
Yes, I tested it with a 100% scale, and everything works correctly. It seems the issue is specific to the 150% scale setting.

**Comment by Vaei** [comment]
I'm not sure I can support this, its likely related to Maya's Python API, and I lack the time to investigate a solution
